EIPS/eip-2935.md

```diff
@@ -103,21 +111,32 @@ push1 0x20
 push0
 return
 
-// revert
+// 0x53: revert
 jumpdest
 push0
 push0
 revert
 
+// 0x57: set op - sstore the input to number-1
+jumpdest
+push0
+calldataload
+push1 0x1
+number
+sub
```

Comment:
you need to mod it by 8192 here, in order to sanitize the input. This has a major drawback though: it means that we can not change the value of the buffer at a later time.

Reply:
I guess it's better to leave it as is then, but it would be a good idea to add a note about it in the next section.

Reply:
fixed/updated